src/plasmapy/utils/decorators/checks.py

```diff
@@ -375,8 +374,7 @@ class CheckUnits(CheckBase):
         equivalencies                  | [DEFAULT `None`] A list of equivalent pairs to
                                          try if
                                        | the units are not directly convertible.
-                                       | (see :mod:`~astropy.units.equivalencies`,
```

Comment:
Since astropy doesn't want the equivalencies.py file to be public, and just to get these objects from the units namespace, I think the best thing to do is just to have the docs point to the documentation page for Equivalencies, which still exists?

Reply:
That's what I had been leaning towards, so let's go with that!